src/auro-select.js

```diff
@@ -26,6 +26,7 @@ import styleCssFixed from './style-fixed-css.js';
  * @prop {String} value - Value selected for the component.
  * @prop {Boolean} error - When attribute is present element shows error state.
  * @prop {Boolean} disabled - When attribute is present element shows disabled state.
+ * @prop {Boolean} checkmark - When true, selected option will show the checkmark.
```

Comment:
This is incorrect. The default should be with the checkmark and the new feature is to remove the checkmark. Update this feature to be `noCheckmark`.